public/css/main.css

```diff
@@ -1005,4 +1005,13 @@ body.darkBG .keynav-target {
 }
 .fs15 span.rm-spacing-tb .int {
   line-height: 1.6rem;
+}
+.sync-progress-bar {
+  animation: glowGreen 3000ms infinite;
+}
```

Comment:
We may wish to limit the width of the progress bar.  It looks a bit silly when the full width of a maximized window.
Perhaps adding to `.progress` way up on line 823 with `width: 480px` or something?